src/views/Portal/components/FormInputs/Radio/index.scss

```diff
@@ -28,22 +29,28 @@
 
   &__inputs {
     display: flex;
-    flex-direction: row;
-    justify-content: flex-start;
-    align-items: center;
+    flex-direction: column;
```

Comment:
We should probably pass in a className into this component to separate `flex: row` vs `flex: column` since other parts of the app use column.